recipes/ComponentsMultiTypeReactHooks/src/Main.purs

```diff
@@ -0,0 +1,136 @@
+module ComponentsMultiTypeReactHooks.Main where
+
+import Prelude
+import Data.Maybe (Maybe(..), fromMaybe)
+import Data.Tuple (fst)
+import Effect (Effect)
+import Effect.Exception (throw)
+import React.Basic.DOM (render)
+import React.Basic.DOM as R
+import React.Basic.DOM.Events (targetValue)
+import React.Basic.Events (EventHandler, handler, handler_)
+import React.Basic.Hooks
+  ( Component
+  , Hook
+  , UseState
+  , component
+  , useState
+  , useState'
+  , (/\)
+  , type (/\)
+  )
+import React.Basic.Hooks as React
+import Web.HTML (window)
+import Web.HTML.HTMLDocument (body)
+import Web.HTML.HTMLElement (toElement)
+import Web.HTML.Window (document)
+
+main :: Effect Unit
+main = do
+  body <- body =<< document =<< window
+  case body of
+    Nothing -> throw "Could not find body."
+    Just b -> do
+      container <- mkContainer
+      render (container unit) (toElement b)
+
+mkContainer :: Component Unit
+mkContainer = do
+  componentA <- mkComponentA
+  componentB <- mkComponentB
+  componentC <- mkComponentC
+  component "Container" \_ -> React.do
+    observedState /\ setObservedState <-
+      useState' { a: Nothing, b: Nothing, c: Nothing }
+    enabledState <- useState false
+    countState <- useState 0
+    inputState <- useInput ""
+    pure
+      $ R.div_
+          [ R.div
+              { className: "box"
```

Comment:
I used this `"box"` class name because the Halogen version uses it. There's no stylesheet for either of them when I run the `make` command, so maybe I'm missing something, or maybe the style is applied in some other setting.

Reply:
Possibly by mistake?